client/ayon_houdini/api/plugin.py

```diff
@@ -334,6 +348,34 @@ class HoudiniInstancePlugin(pyblish.api.InstancePlugin):
     hosts = ["houdini"]
     settings_category = SETTINGS_CATEGORY
 
+    def evalParmNoFrame(self, rop, parm, **kwargs):
+        try:
+            return evalParmNoFrame(rop, parm, **kwargs)
+        except Exception as exc:
+            raise PublishError(
+                f"Failed evaluating parameter '{parm}' on Rop node: {rop.path()}",
+                detail=f"{exc}"
+            )
+    
+    def get_output_parameter(self, node):
+        try:
+            return get_output_parameter(node)
+        except Exception as exc:
```

Comment:
Technically we should only be capturing the `TypeError` here. Right?

Reply:
yes. if the `node.parm(...)` didn't failed. and I think it won't fail.